src/configuration.ts

```diff
@@ -30,6 +30,10 @@ export class Config {
     return this.get<boolean>("enableLanguageServer", false);
   }
```

Comment:
It *can* be, but I need to set it ahead of opening a project without LSP. My issue is not with the LSP being enabled or disabled, it is the pop-ups that LSP is not found. If there is a LSP in PATH in a dev shell, I want to use it. Otherwise, it should ignore the option and suppress that warnings.